composer.json

```diff
@@ -1,7 +1,7 @@
 {
-	"name": "paymentwall/paymentwall-php",
+	"name": "inetbusiness/paymentwall-php",
```

Comment:
@paymentwall-dev somebody tries to change package name, looks like malware commit